app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -22,31 +20,34 @@
 import java.nio.charset.Charset;
 import java.util.List;
 
-import io.ooni.mk.MKCollectorResubmitResults;
-import io.ooni.mk.MKCollectorResubmitTask;
+import io.ooni.mk.MKReporterResults;
+import io.ooni.mk.MKReporterTask;
 import localhost.toolkit.os.NetworkProgressAsyncTask;
 
 public class ResubmitTask<A extends AppCompatActivity> extends NetworkProgressAsyncTask<A, Integer, Boolean> {
+    private MKReporterTask task;
+
     /**
      * Use this class to resubmit a measurement, use result_id and measurement_id to filter list of value
      * {@code new MKCollectorResubmitTask(activity).execute(@Nullable result_id, @Nullable measurement_id);}
      *
      * @param activity from which this task are executed
      */
-    public ResubmitTask(A activity) {
+    public ResubmitTask( A activity) {
```

Comment:
why the space?

Reply:
I believe user error. Will revert this change. Thanks for spotting it!

Reply:
Done!